docs/data/charts/zoom-and-pan/zoom-and-pan.md

```diff
@@ -122,3 +122,75 @@ The `zoom` state is an array of objects that define the zoom state for each axis
 To synchronize zoom between multiple charts, you can control the zoom state.
 
 {{"demo": "ZoomControlled.js"}}
+
+## Zoom configuration
+
+You can have fine-grained control over which interactions are enabled and under which conditions by using the `zoomConfig` prop.
+
+### Interactions
+
+The `zoomConfig` prop allows you to specify which interactions are enabled for zooming and panning:
+
+```jsx
+<BarChartPro
+  zoomConfig={{
+    // Enable both wheel and pinch zoom
+    zoom: ['onWheel', 'onPinch'],
+    // Enable drag panning
+    pan: ['onDrag'],
+  }}
+/>
+```
+
+### Key modifiers
+
+Some interactions allow setting up required keys to be pressed to enable the interaction.
+This can be set up using the `keys` property in the interaction configuration.
+
+```jsx
+<BarChartPro
+  zoomConfig={{
+    // Only zoom when Control key is pressed
+    zoom: [{ type: 'onWheel', keys: ['Control'] }],
+    // Only pan when Shift key is pressed
+    pan: [{ type: 'onDrag', keys: ['Shift'] }],
+  }}
+/>
+```
+
+Available keys include:
+
+- Modifier keys: `'Shift'`, `'Control'`, `'Alt'`, `'Meta'`
+- Special combinations: `'ControlOrMeta'` (useful for cross-platform support)
+- Letter keys: `'a'` through `'z'`
```

Comment:
I've never seen letters being used as a combination for interaction. It's adding a little more complexity for a very nich usage, but why not